src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -148,7 +148,16 @@ export const selectPendingAppointments = createSelector(
 export const selectPastAppointments = createSelector(
   state => state.appointments.past,
   past => {
-    return past?.filter(isValidPastAppointment).sort(sortByDateDescending);
+    if (!past) {
+      return null;
+    }
+
+    const sortedAppointments = past
+      .concat(past)
```

Comment:
This is duplicating all the appointments in the list, looks like

Reply:
@jbalboni regarding the selector, makes sense now. I was thinking with the removal of the express care parts it wasn't an issue but I see the conflict now.

With the duplicates good catch 👀 ! I don't see where the deltas are between the new and old pages and the data that is in state. Will discuss via slack.